tests/open_alchemy/helpers/test_peek/test_peek_extension.py

```diff
@@ -25,8 +26,10 @@ def test_autoincrement_wrong_type():
         ({}, None),
         ({"x-autoincrement": True}, True),
         ({"x-autoincrement": False}, False),
+        ({"x-open-alchemy-autoincrement": True}, True),
+        ({"x-open-alchemy-autoincrement": False}, False),
     ],
-    ids=["missing", "true", "false"],
+    ids=["missing", "true", "false", "true", "false"],
```

Comment:
The only reason that this uses the old ways of `ids` is because I haven't changed to the new way, I use the rule of "if I touch it I need to update it". I'll add this to the `CONTRIBUTING.md`.

I prefer to keep the id with the `purest.param` because it means that you don't have to figure out indexes to know which one is which, this is especially useful for cases where there are many tests.

Reply:
I assume, you mean `pytest.param`, and I'll update the parametrized test cases with the appropriate IDs.

Reply:
> I assume, you mean `pytest.param`, and I'll update the parametrized test cases with the appropriate IDs.

Apple auto-correct doesn't know what `pytest` is 😆